core-libs/setup/ssr/logger/services/prerendering-logger.service.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * SPDX-FileCopyrightText: 2023 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Injectable, inject } from '@angular/core';
+import { LoggerService, WindowRef } from '@spartacus/core';
+import { format } from 'util';
+import { serverLoggerToken } from '../loggers/server-logger';
+
+@Injectable({ providedIn: 'root' })
+export class PrerenderingLoggerService implements LoggerService {
+  ssrLogger = inject(serverLoggerToken);
```

Comment:
[MINOR]

To avoid ambiguity and for consistency, I suggest to rename it to `serverLogger = ...`